src/umap_.jl

```diff
@@ -208,7 +208,7 @@ function initialize_embedding(graph::AbstractMatrix{T}, n_components, ::Val{:spe
         embed = spectral_layout(graph, n_components)
         # expand
         expansion = 10 / maximum(embed)
-        @. embed = (embed*expansion) + (1//10000)*randn(T, size(embed))
+        embed .= (embed*expansion) .+ (1//10000).*randn.(T)
```

Comment:
Thanks for this - I think the only thing is to fuse the multiplication as well, otherwise there's still an allocation.

```suggestion
        embed .= (embed .* expansion) .+ (1//10000) .* randn.(T)
```